docs/docs/development/contributing-boards.md

```diff
@@ -0,0 +1,19 @@
+---
+id: contributing-boards
+title: Community Supported Boards
+---
+
+## About Community Supported Boards
+
+There are many examples of SBCs (small board computers) that hav a passionate community behind them, Jetson Nano for example. These SBCs often have dedicated hardware that can greatly accelerate Frigate's AI and video workloads, but this hardware requires very specific frameworks for interfacing with it.
```

Comment:
I am no native speaker, so bear that in mind.

There are many SBCs (small board computers) that have a passionate community behind them, Jetson Nano for example. These SBCs often have dedicated hardware that can greatly accelerate Frigate's AI and video workloads, but this hardware requires very specific frameworks for interfacing with it.

Reply:
Thanks!